js/index.js

```diff
@@ -40,3 +41,70 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+// const navItem = document.getElementsByTagName('a');
+// navItem.innerHTML = siteContent['nav']['a']
+
+let links = document.querySelectorAll('a');
+links[0].textContent = siteContent.nav['nav-item-1'];
+links[1].textContent = siteContent.nav['nav-item-2'];
+links[2].textContent = siteContent.nav['nav-item-3'];
+links[3].textContent = siteContent.nav['nav-item-4'];
+links[4].textContent = siteContent.nav['nav-item-5'];
+links[5].textContent = siteContent.nav['nav-item-6'];
```

Comment:
Leaving this loop refactor here for reference:
```
let selectedNavLinks = document.querySelectorAll("nav a");
selectedNavLinks.forEach((link, i) => {
  link.innerHTML = siteContent.nav[`nav-item-${i+1}`];
})
```
when using forEach, map, filter, or reduce, the second parameter is always the array's index.